controllers/controllers/services/bindings/managed/controller.go

```diff
@@ -254,6 +279,73 @@ func (r *ManagedBindingsReconciler) finalizeCFServiceBinding(
 	return ctrl.Result{}, nil
 }
 
+func (r *ManagedBindingsReconciler) reconcileSBServiceBinding(ctx context.Context, cfServiceBinding *korifiv1alpha1.CFServiceBinding) (*servicebindingv1beta1.ServiceBinding, error) {
+	sbServiceBinding := r.toSBServiceBinding(cfServiceBinding)
+
+	_, err := controllerutil.CreateOrPatch(ctx, r.k8sClient, sbServiceBinding, func() error {
+		sbServiceBinding.Spec.Name = getSBServiceBindingName(cfServiceBinding)
+
+		return controllerutil.SetControllerReference(cfServiceBinding, sbServiceBinding, r.scheme)
+	})
+	if err != nil {
+		return nil, err
+	}
+
+	return sbServiceBinding, nil
+}
+
+func (r *ManagedBindingsReconciler) toSBServiceBinding(cfServiceBinding *korifiv1alpha1.CFServiceBinding) *servicebindingv1beta1.ServiceBinding {
+	return &servicebindingv1beta1.ServiceBinding{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      fmt.Sprintf("cf-binding-%s", cfServiceBinding.Name),
+			Namespace: cfServiceBinding.Namespace,
+			Labels: map[string]string{
+				korifiv1alpha1.ServiceBindingGUIDLabel:           cfServiceBinding.Name,
+				korifiv1alpha1.CFAppGUIDLabelKey:                 cfServiceBinding.Spec.AppRef.Name,
+				korifiv1alpha1.ServiceCredentialBindingTypeLabel: "app",
+			},
+		},
+		Spec: servicebindingv1beta1.ServiceBindingSpec{
+			Type: korifiv1alpha1.ManagedType,
+			Workload: servicebindingv1beta1.ServiceBindingWorkloadReference{
+				APIVersion: "apps/v1",
+				Kind:       "StatefulSet",
+				Selector: &metav1.LabelSelector{
+					MatchLabels: map[string]string{
+						korifiv1alpha1.CFAppGUIDLabelKey: cfServiceBinding.Spec.AppRef.Name,
+					},
+				},
+			},
+			Service: servicebindingv1beta1.ServiceBindingServiceReference{
+				APIVersion: "korifi.cloudfoundry.org/v1alpha1",
+				Kind:       "CFServiceBinding",
+				Name:       cfServiceBinding.Name,
+			},
+		},
+	}
+}
+
+func getSBServiceBindingName(cfServiceBinding *korifiv1alpha1.CFServiceBinding) string {
+	if cfServiceBinding.Spec.DisplayName != nil {
+		return *cfServiceBinding.Spec.DisplayName
+	}
+
+	return cfServiceBinding.Status.Binding.Name
+}
+
+func isSbServiceBindingReady(sbServiceBinding *servicebindingv1beta1.ServiceBinding) bool {
+	readyCondition := meta.FindStatusCondition(sbServiceBinding.Status.Conditions, "Ready")
```

Comment:
We could simplify the logic around checking for the ready condition by using `meta.IsStatusConditionTrue`, i.e.

```
if !meta.IsStatusConditionTrue(sbServiceBinding.Status.Conditions, "Ready) {
  return false
}

return sbServiceBinding.Generation == sbServiceBinding.Status.ObservedGeneration
```